### ISSUE TITLE
PyPI lib version (0.5.2) doesn't work with python 3.10

### ISSUE THREAD

**Issue by elchupanebrej** [fnpy/fn.py#44]
Installing PyPI version of this lib produces a lot of next messages:

```
Using cached fn.py-0.4.3.tar.gz (32 kB)
  Preparing metadata (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/goloveshkokonstantin/venv/bin/python3.10 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/2g/4nxj2yr14sx8nx_jf8yfkz6m0000gn/T/pip-install-thc9c89p/fn-py_d5536860a1c542198d6eae3fd653f26b/setup.py'"'"'; __file__='"'"'/private/var/folders/2g/4nxj2yr14sx8nx_jf8yfkz6m0000gn/T/pip-install-thc9c89p/fn-py_d5536860a1c542198d6eae3fd653f26b/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/2g/4nxj2yr14sx8nx_jf8yfkz6m0000gn/T/pip-pip-egg-info-n_rplav_
       cwd: /private/var/folders/2g/4nxj2yr14sx8nx_jf8yfkz6m0000gn/T/pip-install-thc9c89p/fn-py_d5536860a1c542198d6eae3fd653f26b/
  Complete output (11 lines):
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/private/var/folders/2g/4nxj2yr14sx8nx_jf8yfkz6m0000gn/T/pip-install-thc9c89p/fn-py_d5536860a1c542198d6eae3fd653f26b/setup.py", line 6, in <module>
      import fn
    File "/private/var/folders/2g/4nxj2yr14sx8nx_jf8yfkz6m0000gn/T/pip-install-thc9c89p/fn-py_d5536860a1c542198d6eae3fd653f26b/fn/__init__.py", line 1, in <module>
      from .stream import Stream
    File "/private/var/folders/2g/4nxj2yr14sx8nx_jf8yfkz6m0000gn/T/pip-install-thc9c89p/fn-py_d5536860a1c542198d6eae3fd653f26b/fn/stream.py", line 9, in <module>
      from .iters import map, range
    File "/private/var/folders/2g/4nxj2yr14sx8nx_jf8yfkz6m0000gn/T/pip-install-thc9c89p/fn-py_d5536860a1c542198d6eae3fd653f26b/fn/iters.py", line 2, in <module>
      from collections import deque, Iterable
  ImportError: cannot import name 'Iterable' from 'collections' (/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/collections/__init__.py)
  ----------------------------------------
```

**Comment by franckrasolo** [comment]
It was [reported](https://github.com/kachayev/fn.py/issues/91) upstream and @lucasew proposed a [fix](https://github.com/kachayev/fn.py/pull/92) that unfortunately never got merged.

**Comment by rodgarci** [comment]
Actually, I think this is already fixed in main with the merge #39, we only would need a new release.
@jacobbridges, this lib is really cool, it would be a pity if we could not use it with python 3.10, could you make a new release please ?

**Comment by vijaybvc1989** [comment]
Can we make a new release please?

**Comment by jacobbridges** [comment]
Apologies to all, life has been busy. I will try to get a release out this evening.

**Comment by jacobbridges** [comment]
@elchupanebrej this issue is resolved in [v0.6.0](https://github.com/fnpy/fn.py/releases/tag/v0.6.0). Please try installing again and apologies again for the delay.
